test/regress_ssl.c

```diff
@@ -638,6 +640,8 @@ wm_acceptcb(struct evconnlistener *listener, evutil_socket_t fd,
 
 	/* Only accept once, then disable ourself. */
 	evconnlistener_disable(listener);
+end: 
+	;
```

Comment:
Adding an empty `end` label is almost always an error, and I'm not sure that it is better then adding a check against NULL in tests